src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,5 +1,127 @@
 package mate.academy
 
+import mate.academy.model.Color
+import mate.academy.service.ColorSupplier
+import kotlin.math.PI
+import kotlin.math.pow
+import kotlin.random.Random
+
+abstract class Figure(val color: Color) {
+    abstract fun area(): Double
+    abstract fun draw()
+}
+
+class Square(private val side: Double, color: Color) : Figure(color) {
+    override fun area(): Double = side * side
+
+    override fun draw() {
+        println("Figure: square, area: ${area()} sq. units, side: $side units, color: $color")
+    }
+}
+
+class Rectangle(private val length: Double, private val width: Double, color: Color) : Figure(color) {
+    override fun area(): Double = length * width
+
+    override fun draw() {
+        println("Figure: rectangle, area: ${area()} sq. units, length:"
+                + " $length units, width: $width units, color: $color")
+    }
+}
+
+class RightTriangle(private val firstLeg: Double, private val secondLeg: Double, color: Color) : Figure(color) {
+    companion object {
+        const val HALF = 0.5
```

Comment:
I see duplication of this line, save it under class
![image](https://github.com/mate-academy/kt-oop-figures/assets/156338484/d6d9804e-f7dd-48da-8230-ab96209e1246)